waspc/data/Generator/templates/react-app/src/operations/counter.js

```diff
@@ -0,0 +1,26 @@
+const counter = new Map()
+
+export function makeCounter(calculateHash) {
```

Comment:
Although this is a pretty general module, I decided it to keep it local for now.